sdk/python/setup.py

```diff
@@ -119,14 +119,17 @@
     "pip-tools",
 ] + GCP_REQUIRED + REDIS_REQUIRED + AWS_REQUIRED
 
-DEV_REQUIRED = ["mypy-protobuf==1.*", "grpcio-testing==1.*"] + CI_REQUIRED
+DEV_REQUIRED = [
```

Comment:
apparently this change doesn't pass python lint, @danilopeixoto

Reply:
Can you please revert it as well?

Reply:
Removing newlines...